js/templates/notification.html

```diff
@@ -13,7 +13,7 @@
                             <%= polyglot.t('NotificationOrderConfirmed') %>
                         </div>
                         <div class="textSize11px floatLeft txt-fade"><%= ob.moment(new Date(ob.timestamp*1000)).locale(window.lang).format('MMM D, h:mm A') %></div>
-                    <% } else if(ob.type == "dispute_close") { %>
```

Comment:
I think the server currently only sends upper case, handling lower case was left in because it was there before, and I didn't want to add a new bug in the 1.1.9 release.